### DISCUSSION TITLE
Panopticon install bump

### DISCUSSION THREAD

**Discussion by Linkedla** [akeeba/panopticon#577] [Q&A (Community Support)] [answered]
Hello, I am trying to install Panopticon, I get the the Create Admin User step, fill in the fields, like i would do installing Joomla, after click next, it simply reloads the page.
I checked the DB, no user gets created.
I have  PHP 8.3.2 and MySQLi 8.0.36

I used Firefox and after Chrome with the same result.

Thank you for your help.

**Comment by nikosdion** [comment]
There must be an error in one of the fields.

**Comment by nikosdion** [comment]
Please do check again with version 1.1.1.

**Comment by gloriadeo** [comment]
Hi, 

I have the same error.  I am unable to setup a user.  The page just resets.

I've tried with 1.1.2 and 1.1.1 and the issue still persists.

**Reply by gloriadeo** [reply]
I just tried with version 1.0.1 and was not able to create a user.

**Reply by nikosdion** [reply]
We know for a fact that both 1.0.1 and 1.1.2 have no problem.

I can only assume that the hosting provider you are using has a database server version which does not fully support JSON fields. This would mean that some of the necessary database tables are not installed, which is why you cannot proceed.

**Reply by Linkedla** [reply, marked as answer]
I has a problem with my server using CloudLinux and lsapi (on my end, bad configuration) i switched to FPM and now is working like a charm...
I am going to love this component!

**Reply by nikosdion** [reply]
Thank you for the feedback and the kind words :) Have a great day!

**Comment by gloriadeo** [comment]
Hi, 

I was able to get past the User page.  I believe that the issue lies with PHP 8.1.  I set my site to PHP 8.2 via the MultiPHP Manager in cPanel and it was able to work just fine.
